### ISSUE TITLE
Autolinking support RN 60+

### ISSUE THREAD

**Issue by sayler8182** [zubricky/react-native-android-keyboard-adjust#28]

**Comment by likern** [comment]
It looks like **autolinking is working out of the box**.
I've updated settings as is said in README, _excluding_ updating **MainActivity.java** (haven't add package manually) and it works just fine.

Otherwise, when I tried to add package manually - I've got error saying that package is registered twice.

**Comment by DnEgorWeb** [comment]
README is really outdated, I opened a PR a year and a half ago to fix a line (but no response), now it should be also corrected for RN 60+. Not so obvious for those who use the package for the very first time.

**Comment by storatus** [comment]
Is this working fine ? I have implemented it in RN 61.5 leaving out the MainActivity.java step, and it seems to be working. Anyone else had some experience ?

**Comment by ssilverberry** [comment]
@storatus Yeah, me too, actually I've tried both options, with all these pointed steps in README (but also skipped MainActivity.java step) also just installed the library without adding anything else in android folders/configs, it seems to be working I mean it changes keyboard adjustment and there are no errors while building an app.

**Comment by czy0729** [comment]
> @storatus Yeah, me too, actually I've tried both options, with all these pointed steps in README (but also skipped MainActivity.java step) also just installed the library without adding anything else in android folders/configs, it seems to be working I mean it changes keyboard adjustment and there are no errors while building an app.

0.63 working fine too

just few steps:

1. npm install --save react-native-android-keyboard-adjust@1.2.0

2. copy `AndroidKeyboardAdjust/*.java`  to   `/android/app/src/main/java/com/your_name/project_name/AndroidKeyboardAdjust`

3. edit `/android/app/src/main/java/com/your_name/project_name/MainApplication.java`
 
```
+ import com.your_name.project_name.AndroidKeyboardAdjust.AndroidKeyboardAdjustPackage;

...

@Override
    protected List<ReactPackage> getPackages() {
      List<ReactPackage> packages = new PackageList(this).getPackages();
      ...
+     packages.add(new AndroidKeyboardAdjustPackage());
      ...
      return packages;
    }

```
that's all

now you can use in your project

```
const AndroidKeyboardAdjust = require('react-native-android-keyboard-adjust')
```

**Comment by karthikNAS** [comment]
> copy AndroidKeyboardAdjust/*.java to /android/app/src/main/java/com/your_name/project_name/AndroidKeyboardAdjust

copy from where?

**Comment by czy0729** [comment]
> > copy AndroidKeyboardAdjust/*.java to /android/app/src/main/java/com/your_name/project_name/AndroidKeyboardAdjust
> 
> copy from where?

https://github.com/zubricky/react-native-android-keyboard-adjust/tree/master/android/src/main/java/net/zubricky/AndroidKeyboardAdjust

**Comment by karthikNAS** [comment]
> > > copy AndroidKeyboardAdjust/*.java to /android/app/src/main/java/com/your_name/project_name/AndroidKeyboardAdjust
> > 
> > 
> > copy from where?
> 
> https://github.com/zubricky/react-native-android-keyboard-adjust/tree/master/android/src/main/java/net/zubricky/AndroidKeyboardAdjust

Still showing following error
A problem occurred evaluating project ':react-native-android-keyboard-adjust'.Could not find method compile() for arguments [com.facebook.react:react-native:+] on object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.

P.S it works if i change compile to implementation in node_modules build.gradle
